### PR TITLE
make migration warning clearer

### PR DESCRIPTION
**Pull request by johanwelgemoed**
## Description

Updating the migration warning we added based on suggestions from support to make less confusing for customers.
The /versioned file contains the text as we would like it to show prior to the upcoming 8.3.1 patch.
Once the patch it released, we will update the /versioned file to have the same text as the updated /next file

## When should this change go live?
asap please

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/operational-guides/update-guide/820-to-830.md [hunk 1]
```diff
@@ -9,8 +9,7 @@ description: "Review which adjustments must be made to migrate from Camunda 8.2.
 The following sections explain which adjustments must be made to migrate from Camunda 8.2.x to 8.3.0 for each component.
 
 :::caution
-Updating Operate, Tasklist, and Optimize from 8.2.x to 8.3.0 will potentially take longer than expected, depending on the data to be migrated.
-Additionally, we identified some bugs that could also prevent the migration from succeeding. These are being addressed and will be available in an upcoming 8.3.1 patch. We suggest not updating until the patch is released.
+Updating Operate, Tasklist, and Optimize from 8.2.x to 8.3.x will potentially take longer than expected, depending on the data to be migrated. We identified some bugs in the 8.3.0 release that could prevent migration from succeeding, which are fixed in 8.3.1. When updating, it is therefore recommended you skip version 8.3.0 and update directly to 8.3.1.
```
Comment (megglos) [inline docs/self-managed/operational-guides/update-guide/820-to-830.md:12]: Let's use the same for next and 8.3 for now, and update both as well once the patches are out to have it consistent. wdyt?
```suggestion
We identified some bugs in 8.3.0 that could prevent the migration from succeeding. These are being addressed and will be available in an upcoming 8.3.1 patch. We suggest not updating until the patch is released.
```

**Review by megglos — DISMISSED** [review]
Thank you for taking action! I have one suggestion to keep both warnings the same for next and 8.3 to have awareness regardless into which version someone looks. We can update both after the patches are available then as well.

**Comment by megglos** [comment]
@camunda/developer-experience apologies for the noise on this, could you take a look on this asap and roll it out, please? Thank you! 🙇

File: docs/reference/announcements.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ Release date: 10th of October 2023
 
 End of maintenance: 9th of April 2025
 
+:::caution
+We identified some bugs in 8.3.0 that could prevent the migration from succeeding. These are being addressed and will be available in an upcoming 8.3.1 patch. We suggest not updating until the patch is released.
```
Comment (christinaausley) [inline docs/reference/announcements.md:14]: Do we need to note this elsewhere in the migration documentation for now?

Comment (megglos) [reply docs/reference/announcements.md:14]: from my perspective not, we have it in the migration guide and the announcements, not aware of any other relevant location 🤔

Comment (christinaausley) [reply docs/reference/announcements.md:14]: Works for me. We can push out a release today to get this on folks' radar.

**Review by christinaausley — APPROVED** [review]

**Comment by christinaausley** [comment]
@johanwelgemoed Feel free to merge when ready 👍

**Pull request merged**